node/tests/RedisClusterClient.test.ts

```diff
@@ -453,4 +453,45 @@ describe("GlideClusterClient", () => {
             client.close();
         },
     );
+
+    it.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
+        `lolwut test routes_%p`,
```

Comment:
I think we can just call it lolwut test since we're testing routes along with general functionality
```suggestion
        `lolwut test_%p`,
```

Reply:
Can we have the duplicated test name since I have the non-cluster tests in RedisClient.test.ts?

Reply:
You can have the same test name, I see other tests in this file that have the same name in the standalone file. Another option would be to name them "lolwut standalone test" and "lolwut cluster test".